source/neuropod/bindings/java/src/main/native/com_uber_neuropod_Neuropod.cc

```diff
@@ -31,6 +31,11 @@ limitations under the License.
 
 using namespace neuropod::jni;
 
+namespace
+{
+jobject toJavaTensorSpecList(JNIEnv *env, const std::vector<neuropod::TensorSpec> &specs);
+}
```

Comment:
`} // namespace`

(Looks like the C++ linters aren't running on this code. I'll look into it and I'm also working on getting lint/static analysis working for the Java/JNI code)